File: uHunt/C++src/easy/EmoogleBalance.cpp
using namespace std;
#include<cstdio>

bool test=true;

int main()
{
    if(test) freopen("../../data/easy/EmoogleBalance.in","r",stdin);

    int numMax, numCeros, actual;
    scanf("%d",&numMax);

    for(int i=1;numMax!=0;i++)
    {
        numCeros=0;
        for(int j=0;j<numMax;j++)
        {
            scanf("%d",&actual);
            if(actual==0) numCeros++;
        }

        printf("Case %d: %d\n",i,(numMax-(2*numCeros)));
        scanf("%d",&numMax);
    }

    return 0;
}

File: uHunt/src/easy/EmoogleBalance.java
package easy;

//ID: 12279
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class EmoogleBalance
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException 
	{
		if(test)br=new BufferedReader(new FileReader(new File("./data/easy/EmoogleBalance.in")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		solve();
		br.close();
	}

	private static void solve() throws IOException
	{
		int numMax=Integer.parseInt(br.readLine().trim()), numCeros=0;
		
		for (int i=1; numMax!=0 ; i++)
		{
			numCeros=0;
			String[] caso=br.readLine().trim().split(" ");
			for (String cadena : caso) if(cadena.equals("0")) numCeros++;
			
			System.out.println("Case "+i+": "+(numMax-(2*numCeros)));
			numMax=Integer.parseInt(br.readLine().trim());
		}
	}
	
}